### ISSUE TITLE
Animation of the Gauge of the type Tank Level doesn't match the fill percentage

### ISSUE THREAD

**Issue by BlueSialia** [FlowFuse/node-red-dashboard#1345]
### Current Behavior

The wave animation only matches the fill percentage at low percentages. Anything above 15-25% and it won't raise anymore. This limit appears to be tied with the browser aspect ratio or resolution.

### Expected Behavior

The wave animation to match the fill percentage. Like in [this article](https://flowfuse.com/blog/2024/08/dashboard-new-layout-widgets-and-gauges/#tank-level) where the wave animation matches the 52%.

### Steps To Reproduce

One can observe at this [live Dashboard](https://dashboard-demos.flowfuse.cloud/dashboard/gauge#tank-level) the behavior I'm describing.

### Environment

- Dashboard version: 1.17.1
- Node-RED version: 4.0.3
- Node.js version: v20.17.0
- npm version: 10.8.2
- Platform/OS: Pop!_OS 22.04 LTS
- Browser: Firefox 130.0.1

### Have you provided an initial effort estimate for this issue?

I am not a FlowFuse team member

**Comment by joepavitt** [comment]
<img width="990" alt="Screenshot 2024-09-30 at 09 44 08" s@BlueSialia c="https://github.com/user-attachments/assets/4de09fe0-0957-473a-860e-354bc8c86e87">

Hi @BlueSialia - can you share a screenshot of what you're seeing please? I'm not able to replicate

**Comment by BlueSialia** [comment]
Here you have it. I've included the HTML element of the Water Tank in the screenshot. Hope it helps.

![Screenshot from 2024-09-30 10-21-25](https://github.com/user-attachments/assets/1358ac91-27f5-499d-a994-89c1fcec06f3)

Also I have a bunch of extensions installed in the browser. Out of all of them, the only ones that I believe can alter the HTML/CSS or block requests in this live Dashboard are uBlock Origin and Dark Reader. I disabled them to test this and the screenshot was taken with them disabled.
